File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.c
#include <stdio.h>

int main(void) {
	char arr[101] = { 0, };
	scanf("%s", arr);
	
	for (int i = 97; i < 123; i++) {
		int j = 0;
		while (arr[j] != 0) {
			if (arr[j] == (char)i) break;

			j++;
		}
		if (arr[j] == (char)i) {
			printf("%d ", j);
		}
		else {
			printf("%d ", -1);
		}

	}

	return 0;
}

File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.py
arr = input()

for i in range(97, 123):
    j = 0
    found = False
    while(j < len(arr)):
        if (chr(i) == arr[j]):
            found = True
            break
        j += 1
    if(found):
        print(j, end=" ")
    else:
        print(-1, end=" ")